linebot/api.py

```diff
@@ -386,6 +387,76 @@ def get_profile(self, user_id, timeout=None):
 
         return Profile.new_from_json_dict(response.json)
 
+    def get_group_summary(self, group_id, timeout=None):
+        """Call get group summary API.
+
+        https://developers.line.biz/en/reference/messaging-api/#get-group-summary
+
+        Gets the group ID, group name, and group icon URL of a group
+        where the LINE Official Account is a member.
+
+        :param str group_id: Group ID
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        :rtype: :py:class:`linebot.models.responses.Group`
+        :return: Profile instance
+        """
+        response = self._get(
+            '/v2/bot/group/{group_id}/summary'.format(group_id=group_id),
+            timeout=timeout
+        )
+
+        return Group.new_from_json_dict(response.json)
+
+    def get_group_members_count(self, group_id, timeout=None):
+        """Call get members in group count API.
+
+        https://developers.line.biz/en/reference/messaging-api/#get-members-group-count
+
+        The number returned excludes the LINE Official Account.
+
+        :param str room_id: Group ID
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        :rtype: :py:class:`linebot.models.responses.Group`
+        :return: Profile instance
+        """
+        response = self._get(
+            '/v2/bot/group/{group_id}/members/count'.format(group_id=group_id),
+            timeout=timeout
+        )
+
+        return Count.new_from_json_dict(response.json)
+
+    def get_room_members_count(self, room_id, timeout=None):
+        """Call get members in room count API.
+
+        https://developers.line.biz/en/reference/messaging-api/#get-members-group-count
+
+        The number returned excludes the LINE Official Account.
+
+        :param str group_id: Group ID
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        :rtype: :py:class:`linebot.models.responses.Group`
+        :return: Profile instance
+        """
+        response = self._get(
+            '/v2/bot/room/{room_id}/members/count'.format(room_id=room_id),
+            timeout=timeout
+        )
+
+        return Count.new_from_json_dict(response.json)
```

Comment:
[imo]
I think that it is redundant to create `Count` model because the response, which contains count property only, is very simple.

Following is one of similar cases, but it returns a response without the new model.

https://developers.line.biz/en/reference/messaging-api/#get-rich-menu-id-of-user
https://github.com/line/line-bot-sdk-python/blob/0f8d285045562ac64c8e5ac8ac9cd4cc9884e60b/linebot/api.py#L690-L709

@okue 
How about this?

Reply:
Thanks! I fixed it.